### PR TITLE
DIT-871: accessibility report changes

### PR DESCRIPTION
**Pull request by SteveSugden**
Change the url end-points in line with accessibility report

### CODE REVIEW

File: app/views/acceptItemInformationList.scala.html [hunk 1]
```diff
@@ -44,6 +39,5 @@
             <li>@messages("acceptItemInformationList.askfor.item2")</li>
         </ul>
 
-        @components.submit_button()
+        @components.button_link(messages("site.continue"), routes.InformationAboutYourItemController.onPageLoad(NormalMode).url)
```
Comment (ed0906) [inline app/views/acceptItemInformationList.scala.html]: I'd make use of the `Navigator` if you can, it enables us to change the page order etc without too much effort

File: app/views/check_your_answers.scala.html [hunk 2]
```diff
@@ -15,27 +15,21 @@
...
 
         @components.heading(messages("checkYourAnswers.heading"))
 
         @for(section <- answerSections if section.containsAnswers) {
             @components.answer_section(section)
         }
 
-        @components.submit_button()
-    }
+        @components.button_link(messages("site.continue"), routes.DeclarationController.onPageLoad(NormalMode).url)
```
Comment (ed0906) [inline app/views/check_your_answers.scala.html]: Try use the `Navigator` if you can

File: conf/messages.en [hunk 3]
```diff
@@ -41,7 +41,7 @@ beforeYouStart.paragraph2 = You need to know:
 beforeYouStart.paragraph2.item1 = how it is made including a list of ingredients or parts
 beforeYouStart.paragraph2.item2 = what it is used for and how it works
 beforeYouStart.paragraph2.item3 = how it is packaged
-beforeYouStart.paragraph3 = If relevant, you''ll also need to:
+beforeYouStart.paragraph3 = If relevant, you’ll also need to:
```
Comment (ed0906) [inline conf/messages.en]: I think you need the `''` or it doesn't render the `'`

File: conf/messages.en [hunk 4]
```diff
@@ -75,7 +75,7 @@ unauthorised.intro2 = CDS is still in development and does not yet offer all ser
 unauthorised.warn_p1 = Once you have confirmation of CDS access, you’ll need to return to this page and
 unauthorised.warn_link = restart your application.
 unauthorised.warn_p2 = If you still cannot start the application, you may have to try again once you have received a confirmation email from CDS.
-unauthorised.get.eori_p1 = To get access to CDS you''ll need an
+unauthorised.get.eori_p1 = To get access to CDS you’ll need an
```
Comment (ed0906) [inline conf/messages.en]: I think you need the '' or it doesn't render the '

File: conf/messages.en [hunk 5]
```diff
@@ -94,14 +94,14 @@ registeredAddressForEori.headingNoEORI = What is your EORI number and what are t
 registeredAddressForEori.note = The details you enter must match the exact format of the registered EORI address
 registeredAddressForEori.eori = EORI number
 registeredAddressForEori.eori.hint = The number starts with a country code, for example GB for Great Britain, and is then followed by upto 15 characters.
-registeredAddressForEori.businessName = Business, organisation or individual''s name
+registeredAddressForEori.businessName = Business, organisation or individual’s name
```
Comment (ed0906) [inline conf/messages.en]: I think you need the '' or it doesn't render the '

File: conf/messages.en [hunk 6]
```diff
@@ -94,14 +94,14 @@ registeredAddressForEori.headingNoEORI = What is your EORI number and what are t
...
 registeredAddressForEori.addressLine1 = Address line 1
 registeredAddressForEori.townCity = Town or city
 registeredAddressForEori.postcode = Postcode
 registeredAddressForEori.country = Country
 registeredAddressForEori.checkYourAnswersLabel = Your registered details
 registeredAddressForEori.error.eori.required = Enter your EORI
-registeredAddressForEori.error.businessName.required = Enter a business, organisation or individual''s name
+registeredAddressForEori.error.businessName.required = Enter a business, organisation or individual’s name
```
Comment (ed0906) [inline conf/messages.en]: I think you need the '' or it doesn't render the '

File: conf/messages.en [hunk 7]
```diff
@@ -125,19 +125,19 @@ selectApplicationType.checkYourAnswersLabel = Is this a new application or a ren
 selectApplicationType.error.required = Select application type
 
 registerBusinessRepresenting.title =  Registered EORI details - GOV.UK
-registerBusinessRepresenting.heading = What are the registered details of the business, organisation or individual you''re representing?
+registerBusinessRepresenting.heading = What are the registered details of the business, organisation or individual you’re representing?
 registerBusinessRepresenting.note = The details you enter must match the exact format of their registered EORI address
 registerBusinessRepresenting.eoriNumber = EORI number
 registerBusinessRepresenting.eoriNumber.hint = The number starts with a country code, for example GB for Great Britain, and is then followed by up to 15 characters.
-registerBusinessRepresenting.businessName = Business, organisation or individual''s name
+registerBusinessRepresenting.businessName = Business, organisation or individual’s name
```
Comment (ed0906) [inline conf/messages.en]: I think you need the '' or it doesn't render the '

File: conf/messages.en [hunk 8]
```diff
@@ -125,19 +125,19 @@ selectApplicationType.checkYourAnswersLabel = Is this a new application or a ren
 selectApplicationType.error.required = Select application type
 
 registerBusinessRepresenting.title =  Registered EORI details - GOV.UK
-registerBusinessRepresenting.heading = What are the registered details of the business, organisation or individual you''re representing?
+registerBusinessRepresenting.heading = What are the registered details of the business, organisation or individual you’re representing?
```
Comment (ed0906) [inline conf/messages.en]: I think you need the '' or it doesn't render the '

File: conf/messages.en [hunk 9]
```diff
@@ -125,19 +125,19 @@ selectApplicationType.checkYourAnswersLabel = Is this a new application or a ren
...
+registerBusinessRepresenting.businessName = Business, organisation or individual’s name
 registerBusinessRepresenting.addressLine1 = Address line 1
 registerBusinessRepresenting.town = Town or city
 registerBusinessRepresenting.postCode = Postcode
 registerBusinessRepresenting.country = Country
 registerBusinessRepresenting.checkYourAnswersLabel =  Their registered details
 registerBusinessRepresenting.error.eoriNumber.required = Enter their EORI number
 registerBusinessRepresenting.error.eoriNumber.format = Enter their EORI number in the correct format
-registerBusinessRepresenting.error.businessName.required = Enter a business, organisation or individual''s name
+registerBusinessRepresenting.error.businessName.required = Enter a business, organisation or individual’s name
```
Comment (ed0906) [inline conf/messages.en]: I think you need the '' or it doesn't render the '

File: conf/messages.en [hunk 10]
```diff
@@ -302,7 +302,7 @@ legalChallenge.heading = Do you know if there have been any problems classifying
 legalChallenge.yesLegalChallenge = Yes
 legalChallenge.noLegalChallenge = No
 legalChallenge.checkYourAnswersLabel = Do you know of any problems with the classification of the item?
-legalChallenge.error.required = Select if you are aware of any problems with the item''s classification
+legalChallenge.error.required = Select if you are aware of any problems with the item’s classification
```
Comment (ed0906) [inline conf/messages.en]: I think you need the '' or it doesn't render the '

File: conf/messages.en [hunk 11]
```diff
@@ -355,13 +355,13 @@ confirmation.download.link.PDF = Download a PDF of the application
 confirmation.download.link.accessibleText = Download an accessible text file of the application
 
 uploadWrittenAuthorisation.title = Upload written authorisation - GOV.UK
-uploadWrittenAuthorisation.heading = Upload written authorisation from the business, organisation or individual you''re representing
+uploadWrittenAuthorisation.heading = Upload written authorisation from the business, organisation or individual you’re representing
```
Comment (ed0906) [inline conf/messages.en]: I think you need the '' or it doesn't render the '

File: conf/messages.en [hunk 12]
```diff
@@ -355,13 +355,13 @@ confirmation.download.link.PDF = Download a PDF of the application
...
 uploadWrittenAuthorisation.label = Upload a file
 uploadWrittenAuthorisation.label.replace = Replace file
 uploadWrittenAuthorisation.checkYourAnswersLabel =  Written authorisation
 uploadWrittenAuthorisation.error.selectFile = You must select a file, you cannot continue without uploading written authorisation
-uploadWrittenAuthorisation.error.size = Your document will not upload because it''s bigger than 10MB
-uploadWrittenAuthorisation.error.fileType = Your document will not upload because it''s not a valid file type
```
Comment (ed0906) [inline conf/messages.en]: I think you need the '' or it doesn't render the '

File: conf/messages.en [hunk 13]
```diff
@@ -355,13 +355,13 @@ confirmation.download.link.PDF = Download a PDF of the application
...
+uploadWrittenAuthorisation.error.size = The selected file must be smaller than 10MB
+uploadWrittenAuthorisation.error.fileType = Your document will not upload because it’s not a valid file type
```
Comment (ed0906) [inline conf/messages.en]: I think you need the '' or it doesn't render the '

File: app/views/acceptItemInformationList.scala.html (hunk 1 above)
Comment (SteveSugden) [reply app/views/acceptItemInformationList.scala.html]: Can't find an example of a page doing this directly.  Can the Navigator be injected into the page?  And what would it do to turn a `UserAnswers => Call` into a url string?

Could go back to calling another GET endpoint on the Controller and having that re-direct (like the POST used to do).   Is that worth it?

Comment (ed0906) [reply app/views/acceptItemInformationList.scala.html]: `Call.url` converts a Call to a URL String

Up to you, if its awkward to do there's no need. We just lose a bit of flexibility if we don't use it

**Pull request merged**